internal/api/api.go

```diff
@@ -71,6 +71,13 @@ func Start(cfg *config.Config) error {
 		logger.Info("disabling access logs for /healthcheck")
 	}
 	accessMiddleware := func(c *gin.Context) {
+		// Skip logging for specified paths
+		for _, path := range skipPaths {
+			if c.Request.URL.Path == path {
+				c.Next()
```

Comment:
Do we need this? The docs say this is a way to short-circuit and run the next handler before returning to this one, but I can't see where we would want to return to this one.

Reply:
It makes sense because we want to skip it entirely. We don't need to handle it. It makes sense to remove c.Next()